spec/models/ext_management_system_spec.rb

```diff
@@ -725,33 +725,36 @@
     it "queues up destroy" do
       described_class.destroy_queue([ems.id, ems2.id])
 
-      expect(MiqQueue.count).to eq(2)
-      expect(MiqQueue.pluck(:instance_id)).to match_array([ems.id, ems2.id])
+      expect(MiqQueue.where(:method_name => "orchestrate_destroy").count).to eq(2)
+      expect(MiqQueue.where(:method_name => "orchestrate_destroy").pluck(:instance_id)).to match_array([ems.id, ems2.id])
     end
   end
 
   context "#destroy_queue" do
+    before       { Zone.seed }
```

Comment:
This seeds the maintenance_zone so that the `pause!` method passes validation (otherwise fails with zone being nil)

Reply:
That weird considering we pass the zone to the ems server - and local_miq_server should create the zone under the covers.  Unless it's to create the maintenance zone?

Reply:
It is to create the maintenance zone

Reply:
Without this pause! fails because Zone.maintenance_zone returns nil

We do this (Zone.seed) everywhere we need the maintenance zone in specs